lib/colorls/flags.rb

```diff
@@ -199,6 +200,9 @@ def add_long_style_options(options)
         long_style_options[:hard_links_count] = false
       end
       long_style_options = get_long_style_symlink_options(options, long_style_options)
+      options.on('--nh', '--non-human-readable', 'show file sizes in bytes only') do
```

Comment:
Not sure it's really worth adding an abbreviation for the long option, since you can always just use a prefix of a long option when it is unambiguous. So `--non` would just work as well as `--nh`. That's only one character more...